langchain4j/src/test/java/dev/langchain4j/service/StreamingAiServicesWithToolsWithoutMemoryIT.java

```diff
@@ -158,6 +156,8 @@ void should_execute_multiple_tools_sequentially_then_answer() throws Exception {
         verifyNoMoreInteractions(calculator);
 
         ArgumentCaptor<List<ChatMessage>> sendMessagesCaptor = ArgumentCaptor.forClass(List.class);
+
+        // TODO failing: was 2
```

Comment:
fishy

Reply:
@langchain4j can you run the test using GPT_3_5_TURBO_0613 to check if its failing here? They don't accept new users so I get an error that the model does not exist.

Reply:
This test is green for me for `gpt-3.5-turbo-0613`

Reply:
@langchain4j As they don't accept new users for the model anyone new will get an error and failing test. We keep that way? If keeping it I think we can close the PR, if we are gonna change it looks okay to me changing to 2 for the reason above.

Reply:
Ok, let's drop `GPT_3_5_TURBO_0613`. What about using `GPT_4O_MINI` and explicitly setting `parallelToolCalls=false` in this test?